internal/app/vodtester/vodtester_app.go

```diff
@@ -44,56 +45,76 @@ func NewVodTester(gctx context.Context, opts VodTesterOptions) IVodTester {
 func (vt *vodTester) Start(fileName string, vodImportUrl string, taskPollDuration time.Duration) (int, error) {
 	defer vt.cancel()
 
-	hostName, _ := os.Hostname()
-	assetName := fmt.Sprintf("vod_test_asset_%s_%s", hostName, time.Now().Format("2006-01-02T15:04:05Z07:00"))
+	eg, _ := errgroup.WithContext(vt.ctx)
 
-	importAsset, err := vt.importFromUrlTester(vodImportUrl, taskPollDuration, assetName)
+	eg.Go(func() error {
 
-	if err != nil {
-		glog.Errorf("Error importing asset from url=%s err=%v", vodImportUrl, err)
-		return 0, fmt.Errorf("error importing asset from url=%s: %w", vodImportUrl, err)
-	}
+		hostName, _ := os.Hostname()
+		assetName := fmt.Sprintf("vod_test_asset_%s_%s", hostName, time.Now().Format("2006-01-02T15:04:05Z07:00"))
```

Comment:
WDYT of giving specific names for these, not only "test_asset"

This could be `vod_test_upload_url_...` instead (can ommit the `asset` its redundant)

Others would be `vod_test_upload_direct` and `vod_test_upload_resumable`